indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -374,12 +375,16 @@ def _checkProtection(self):
             raise Forbidden
 
     def _process(self):
+        pdf_class = RegistrantsListToBadgesPDF
```

Comment:
why not set this in an `else:` block below? seems clearer than setting it here and then having if/elif to change it.